crates/ruma-events/tests/it/mod.rs

```diff
@@ -1,5 +1,3 @@
-#![cfg(feature = "events")]
```

Comment:
I believe these tests don't run because this file should be called `main.rs`, not `mod.rs`.